$OpenBSD$

patch away boost related -fno_exceptions + handle sqlite3 in base

--- CMakeLists.txt.orig	Sat Oct  6 02:46:53 2012
+++ CMakeLists.txt	Tue Oct 23 17:48:32 2012
@@ -145,6 +145,8 @@ macro_ensure_out_of_source_build("Compiling Calligra i
 ############################
 ###########################
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${KDE4_ENABLE_EXCEPTIONS}")
+
 find_package(Perl REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(PNG REQUIRED)
@@ -521,9 +523,9 @@ if(BUILD_kexi AND SHOULD_BUILD_KEXI)
     ## Test for sqlite
     ##
     set(SQLITE_MIN_VERSION 3.6.16)
-    set(SQLITE_RECOMMENDED_VERSION 3.7.12)
+    set(SQLITE_RECOMMENDED_VERSION 3.7.13)
     set(SQLITE_LOAD_EXTENSION_REQUIRED ON)
-    find_package(CalligraSqlite 3.6.16)
+    find_package(Sqlite 3.7.13)
     macro_log_feature(SQLITE_FOUND "SQLite" "Embedded SQL database engine with enabled extensions loading"
                       "http://www.sqlite.org" FALSE "${SQLITE_MIN_VERSION}"
                       "Required by Kexi for its default database driver.")
